i18n/es.toml

```diff
@@ -18,9 +18,14 @@ other = "Buscar en este sitio…"
 [ui_in]
 other = "en"
 
-# Used in sentences such as "All Tags"
-[ui_all]
-other = "todo(a)"
+# Phrases for tags
+[ui_see_all_tags]
+other = "See all tags"
```

Comment:
Eventually we have to translate this entry

Reply:
@electrocucaracha  can you create an issue to track this?

Reply:
Done in #1748